sdk/keyvault/azure-keyvault-secrets/src/samples/java/com/azure/security/keyvault/secrets/BackupAndRestoreOperationsAsync.java

```diff
@@ -38,7 +39,8 @@ public static void main(String[] args) throws IOException, InterruptedException,
         // Let's create secrets holding storage account credentials valid for 1 year. if the secret
         // already exists in the key vault, then a new version of the secret is created.
         secretAsyncClient.setSecret(new Secret("StorageAccountPassword", "f4G34fMh8v-fdsgjsk2323=-asdsdfsdf")
-            .setExpires(OffsetDateTime.now().plusYears(1)))
+            .setProperties(new SecretProperties()
+            .setExpires(OffsetDateTime.now().plusYears(1))))
```

Comment:
This seems less user friendly rather than more user friendly?

Reply:
Composition change is across all languages.
To support this, setters are idiomatic and logical in Java.
Pros of composition, list* and updateSecretProperties API deals with SecretProperties object now instead of SecretBase, no ambiguity between Secret and SecretBase anymore.